ir-engine/etherealengine-archive
# IR-2723: Zendesk widget in viewer app & custom button

## Summary

### Custom button to open zendesk chat in studio

![Screen Recording 2024-07-01 at 1 27 42 p m](https://github.com/EtherealEngine/etherealengine/assets/33644205/e3ca96e7-a9cf-4e06-8b57-41e80f13157a)

### Custom button to open zendesk chat in viewer app 

![Screen Recording 2024-07-01 at 6 54 28 p m](https://github.com/EtherealEngine/etherealengine/assets/33644205/1ba3844e-e74f-4c02-ae0e-eceb4bc0ccd4)


Threads with UI/UX team in figma:
- https://www.figma.com/design/A9pA53WkrvRMz8krc9XEiY?node-id=1637-914#858964434
- https://www.figma.com/design/A9pA53WkrvRMz8krc9XEiY?node-id=781-3162#855825043

## Subtasks Checklist

## Breaking Changes

## References
closes #_insert number here_

## QA Steps

#### Test in studio 
1. set the following environment variables in your local environment: `VITE_ZENDESK_ENABLED`, `VITE_ZENDESK_KEY`    
2. log in 
3. Go to the studio
4. Click on the help button on the lower right corner, it should open the zendesk chat

#### Test in client  (viewer app) 
1. set the following environment variables in your local environment: `VITE_ZENDESK_ENABLED`, `VITE_ZENDESK_KEY`    
2. log in 
3. Go to any location 
4. Click on the profile button
5. Click on the help button on the upper right corner, it should open the zendesk chat
